src/features/runner.ts

```diff
@@ -51,106 +50,101 @@ export class AnsiblePlaybookRunProvider {
         vscode.window.showErrorMessage(
           `No Ansible playbook file has been specified
                     to be executed with ansible-${
-  useNavigator
-    ? 'navigator'
-    : 'playbook'
-}.`,
+  useNavigator ? 'navigator' : 'playbook'
+}.`
         );
         return;
       }
       console.debug(
         `Got a request to run ansible-${
-          useNavigator
-            ? 'navigator'
-            : 'playbook'
+          useNavigator ? 'navigator' : 'playbook'
         } against`,
-        playbookFsPath,
+        playbookFsPath
```

Comment:
Always keep trailing commas.

Reply:
Irrelevant because all changes are made by prettier, on save. There is nothing configurable about it.